membrane_curvature/base.py

```diff
@@ -15,33 +15,35 @@
 from membrane_curvature.surface import get_z_surface
 from membrane_curvature.curvature import mean_curvature, gaussian_curvature
 
-
-from MDAnalysis.analysis.base import AnalysisBase
 import MDAnalysis as mda
+from MDAnalysis.analysis.base import AnalysisBase
 
+import logging
+
+logger = logging.getLogger("MDAnalysis.analysis.density")
```

Comment:
name your logger differently ... something like
```suggestion
logger = logging.getLogger("MDAnalysis.MDAKit.membrane_curvature")
```
I started your logger's name with "MDAnalysis" so that output gets added to the default logger that can be started with
```python
import MDAnalysis
MDAnalysis.start_logging()
```
and whose root logger is named "MDAnalysis". Hence any logger named "MDAnalysis.*" will also log under it.

I suggest "MDAnalysis.MDAKit" to define a namespace that all MDAKits could be using but we haven't really thought about this so this could definitely be subject to change. Maybe @IAlibay @lilyminium @richardjgowers @fiona-naughton have opinions.

In any case, for right now, pretty much anything is ok that's not "density" ;-).